### DISCUSSION TITLE
Ownership verification failed

### DISCUSSION THREAD

**Discussion by itzsurbhisharma** [community/community#119769] [Repositories]
### Select Topic Area

General

### Body

I am trying to add my repository to the Google Search Console. I am adding the .html file to my repository as very but when I click verify. The ownership Verification is failing. Please Help.

**Comment by queenofcorgis** [comment]
Hey @itzsurbhisharma thanks for coming on down to the Community 😃 

Have you checked out [this discussion](https://github.com/orgs/community/discussions/42375), there are a lot of great tips for this!
